tests/phpunit/elementor/modules/atomic-widgets/test-atomic-svg.php

```diff
@@ -15,16 +15,16 @@ class Test_Atomic_Svg extends Elementor_Test_Base {
 	public function setUp() : void {
 		parent::setUp();
 
-		add_filter( 'wp_get_attachment_image_src', function( $image, $attachment_id, $size ) {
+		add_filter( 'get_attached_file', function( $path, $attachment_id ) {
 			if ( $attachment_id === 123 ) {
-				return [ self::TEST_RESOURCES_DIR . 'test.svg' ];
+				return self::TEST_RESOURCES_DIR . 'test.svg';
 			}
 
-			return $image;
-		}, 10, 3 );
+			return $path;
+		}, 10, 2 );
```

Comment:
```suggestion
		} );
```

Reply:
I need the second arg, so must set the number of args

Reply:
For some reason I thought 10,2 is the default (which doesn't make sense actually...)

never mind